### ISSUE TITLE
Disable spacebar from accessibility

### ISSUE THREAD

**Issue by MuttakinHasib** [tailwindlabs/headlessui#1442]
Sorry I need to use the Menu library with a form. My problem is that on the form field I cannot use the spacebar. Is there any way to disable spacebar from accessibility?
